utils/src/server/rpc_module.rs

```diff
@@ -180,6 +180,17 @@ impl Methods {
 		}
 	}
 
+	/// Test helper to call a method on the `RPC module` without having to spin a server up.
+	///
+	/// Converts the params to a stringified array for you if it's not already serialized to a sequence.
+	pub async fn test_call<T: Serialize>(&self, method: &str, params: T) -> Option<String> {
```

Comment:
Nitpick: `test_call` makes it sound like this should only be used for tests (perhaps true, but there may be other uses, and it's available all of the time), but I wonder whether something like `local_call` would be a better description of what it does (allow you to call a method locally rather than over the network interface)?

Reply:
It should "only" used for tests to unit tests your `RPC module`, see https://github.com/paritytech/jsonrpsee/pull/458#discussion_r706087979

So it's a test helper so might be a little weird but I think it's useful and I didn't want to introduce a `test-helper` feature it.

Does it make sense?

Reply:
Ah interesting, ok! I guess I could see uses for this beyond just testing. For example, if you wanted to put a RESTish API or GraphQL or something on top of your RPC interface, you could use `call_with`/`test_call` to make those calls without needing to go via the network interface.

But anyway, I'm happy either way really if you prefer `test_call` :)

Reply:
ok, didn't realize that use-case renamed it now thanks